Components/LandingPage.jsx

```diff
@@ -0,0 +1,84 @@
+import { View, Button, StyleSheet, Text, TouchableOpacity } from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+import React from 'react';
+import { Video } from 'expo-av';
+
+const LandingPage = ({ navigation }) => {
+  return (
+    <LinearGradient
+      colors={['#6a11cb', '#2575fc']}
```

Comment:
I think the landing page is pretty blue. I'd suggest a predominantly white landing page with blue accents. The blue makes certain text harder to read.